src/actions/helpers/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const ETHEREUM_API_ID = "ethereum";
+export const ERC20_PRICE_ENDPOINT =
+  "https://api.coingecko.com/api/v3/simple/token_price/ethereum?";
```

Comment:
Should we consider a fallback price api?

Reply:
That's a fair concern, we can add a story to also check the CoinMarketCap API for this value if needed.